### PR TITLE
Adding psycopg2 driver-examples tests

### PR DESCRIPTION
**Pull request by Sfurti-yb**
Added scripts as per the given template to run psycopg2 smart driver examples as part of the jenkins pipeline.

### CODE REVIEW

File: psycopg2/start.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+./tear-down.sh
+
+# Print summary
+echo "Returning $SUCCESS"
+summary="FAIL"
+if [[ "$SUCCESS" == "0" ]]; then
+  summary="PASS"
+fi
+printf '%s\n' $summary
+printf '%s\n' "------------- END new_tool run ------------------"
```
Comment (ashetkar) [inline psycopg2/start.sh]: `END new_tool run` -> `END psycopg2 run`

File: psycopg2/run-app.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+source $WORKSPACE/environments/psycopg2-test/bin/activate
+
+pip install psycopg2-yugabytedb
+
+export YB_PATH=$YUGABYTE_HOME_DIRECTORY
+
+EXIT_STATUS=0
+
+python3 test_uniformloadbalancer.py || EXIT_STATUS=$?
```
Comment (ashetkar) [inline psycopg2/run-app.sh:28]: This should `&&` instead of `||`?

File: psycopg2/start.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+printf '%s\n' "------------- START new_tool run ------------------"
```
Comment (ashetkar) [inline psycopg2/start.sh]: `START new_tool run` -> `START psycopg2 run`

File: psycopg2/run-app.sh (hunk 2 above)
Comment (ashetkar) [reply psycopg2/run-app.sh:28]: May be this is right.

Comment (Sfurti-yb) [reply psycopg2/run-app.sh:28]: Confirmed that this is correct.

**Pull request merged**